tests/e2e/utils/index.js

```diff
@@ -17,3 +17,4 @@ export { switchEditorModeTo } from './switch-editor-mode-to';
 export { uploadMedia } from './upload-media';
 export { getBlocksOnPage } from './get-blocks-on-page';
 export { goToPreviousPage } from './go-to-previous-page';
+export { wpDataSelect } from './wp-data-select';
```

Comment:
Just to be sure, this util already existed, it just wasn't included in the index and had only been directly imported previously, right?

Reply:
> Just to be sure, this util already existed, it just wasn't included in the index and had only been directly imported previously, right?

Yes and no. It wasn't included in the index, but imported by some other util. Not directly imported by a test.